packages/LSOnline/modules/utils/Helpers.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
+
+const upperString = string => string.toLowerCase().replace(/(^| )(\w)/g, s => s.toUpperCase());
```

Comment:
in the future this file is going to be "god" file. we can't group many functionalities in single place because we are creating a lot of dependencies in code.

Reply:
We need to separate these helpers in the next routine code refactoring.